recipes/sqlite3/all/conanfile.py

```diff
@@ -174,10 +174,10 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "SQLite3")
         self.cpp_info.names["cmake_find_package"] = "SQLite"
         self.cpp_info.names["cmake_find_package_multi"] = "SQLite"
-        self.cpp_info.set_property("cmake_target_name", "SQLite")
+        self.cpp_info.set_property("cmake_target_name", "SQLite::SQLite")
         self.cpp_info.components["sqlite"].names["cmake_find_package"] = "SQLite3"
         self.cpp_info.components["sqlite"].names["cmake_find_package_multi"] = "SQLite3"
-        self.cpp_info.components["sqlite"].set_property("cmake_target_name", "SQLite3")
+        self.cpp_info.components["sqlite"].set_property("cmake_target_name", "SQLite::SQLite3")
         self.cpp_info.components["sqlite"].builddirs.append(self._module_subfolder)
         self.cpp_info.components["sqlite"].build_modules["cmake_find_package"] = [self._module_file_rel_path]
```

Comment:
how tu support in CMakeDeps these custom CMake variables defined in this module?

Reply:
I guess we can add
```
self.cpp_info.set_property("cmake_build_modules", [build_module_for_cmakedeps])
```

Reply:
So yes maybe adding `self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])` works with `self.cpp_info.set_property("cmake_find_mode", "both")`

Reply:
It has just been included in https://github.com/conan-io/conan/pull/10227